client/extensions/woocommerce/app/products/product-header.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import Button from 'components/button';
+import StickyPanel from 'components/sticky-panel';
+
+const ProductHeader = ( { onTrash, onDuplicate, onSave, translate } ) => {
+	const trashButton = onTrash &&
+		<Button borderless onClick={ onTrash }><Gridicon icon="trash" /></Button>;
+
+	const duplicateButton = onDuplicate &&
+		<Button onClick={ onDuplicate }>{ translate( 'Duplicate' ) }</Button>;
```

Comment:
Hi! I've found a possible matching string that has already been translated **24** times:
`translate( 'Skipping duplicate' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).